gnupg/_meta.py

```diff
@@ -419,28 +418,27 @@ def _homedir_setter(self, directory):
                       % _util._conf)
             directory = _util._conf
 
-        hd = _parsers._fix_unsafe(directory)
```

Comment:
I'd strongly prefer to fix unsafe strings as early as possible to prevent potential problems while working with them later. (For example, this is part of [what makes this library not vulnerable to SigSpoof](https://blog.patternsinthevoid.net/pretty-bad-protocolpeople.html).)